obspy/taup/tests/test_tau.py

```diff
@@ -107,6 +107,28 @@ def test_p_iasp91_manual(self):
         self.assertAlmostEqual(p_arrival.purist_distance, 35.00, 2)
         self.assertEqual(p_arrival.purist_name, "P")
 
+    def test_p_iasp91_geo_manua(self):
```

Comment:
All tests that can only work if `geographiclib` is installed should be skipped if it is not installed, see e.g. https://github.com/obspy/obspy/blob/master/obspy/geodetics/tests/test_util_geodetics.py#L60